tutorial/tracking/model-based/generic/tutorial-mb-generic-tracker-full.cpp

```diff
@@ -299,9 +299,12 @@ int main(int argc, char **argv)
         vpMe me;
         me.setMaskSize(5);
         me.setMaskNumber(180);
+        me.setInitRange(-1); // To use the default values defined for each type of primitive
         me.setRange(8);
         me.setLikelihoodThresholdType(vpMe::NORMALIZED_THRESHOLD);
         me.setThreshold(20);
+        me.setMinThreshold(-1); // To deactivate automatic thresholding
+        me.setThresholdMarginRatio(-1.); // To deactivate automatic thresholding
         me.setMu1(0.5);
```

Comment:
Thanks to update the doc of `vpMe::setThresholdMarginRatio()` to indicate that this parameter allows automatic thresholding + when the value is -1 it is disabled, and precise how the threshold is set: when set to 1, we use the current value of the contrast (grey level) as threshold (if I'm right). When the value is set to 0.5...